### PR TITLE
Adam Fofana - Homework 01 (Redo)

### PR DESCRIPTION
**Pull request by Adam-Fofana**
I've had some experience with R in the past, and I already had RStudio on my machine, so most of my focus regarding this assignment was getting familiar with Git/Github. R Markdown is pretty handy.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* You completed all the HW requirements. Great job :)
* I agree that Rmd is really nice. Definitely useful in many context.

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# Adam Fofana
+
+![Alt text is fun](IMG_2520.jpg)
```
Comment (CommodoreDave) [inline README.md:3]: Go Terps!

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+## Education
+BA in Government and Politics with a Minor in Linguistics from the University of Maryland (May 2019)
+
+MA in International Relations from the University of Chicago (Expected June 2020)
+
+## Academic Interests
+1. Economic Integration in West Africa
+2. International Organizations
+3. Public Transportation
```
Comment (CommodoreDave) [inline README.md:15]: As it concerns Chicago, are you #TeamMetra or #TeamCTA for getting downtown?

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* You completed all the HW requirements. Great job :)
* I agree that Rmd is really nice. Definitely useful in many context.

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+4. Urbanism
+
+## Personal Interests
+1. Running (**>800 meters**)
+2. Comicbooks
+3. Marvel Movies
+4. Science Fiction
```
Comment (CommodoreDave) [inline README.md:22]: Very cool! (There's a comicbook shop in Hyde Park called First Aid Comics- the more you know!)

**Review by CommodoreDave — COMMENTED** [review]
Great execution of the assignment! Perhaps you could have mixed it up with the formatting a bit more, but the final product has a clean and neat appearance to it. Could have added a link to really up the ante, but all of the other suggested elements are in there. Good job!

**Comment by Adam-Fofana** [comment]
I live near a Metra station, so I've been using that so far.

On Oct 8, 2019 4:04 PM, CommodoreDave <notifications@github.com> wrote:

@CommodoreDave commented on this pull request.

________________________________

In README.md<https://github.com/cfss-fa19/hw01/pull/4#discussion_r332732704>:

> \ No newline at end of file
+# Adam Fofana
+
+Alt text is fun
+
+
+
+## Education
+BA in Government and Politics with a Minor in Linguistics from the University of Maryland (May 2019)
+
+MA in International Relations from the University of Chicago (Expected June 2020)
+
+## Academic Interests
+1. Economic Integration in West Africa
+2. International Organizations
+3. Public Transportation


As it concerns Chicago, are you #TeamMetra or #TeamCTA for getting downtown?

**Comment by Adam-Fofana** [comment]
I was there yesterday!

On Oct 8, 2019 4:07 PM, CommodoreDave <notifications@github.com> wrote:

@CommodoreDave commented on this pull request.

________________________________

In README.md<https://github.com/cfss-fa19/hw01/pull/4#discussion_r332733834>:

> +2. Comicbooks
+3. Marvel Movies
+4. Science Fiction


Very cool! (There's a comicbook shop in Hyde Park called First Aid Comics- the more you know!)

**Review by adamberger20 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Excellent, concise presentation! I feel like I learned a lot about you in very few words.
* All the headings, special text, and lists turned out well.
*Only problem I had is the picture you posted at the very top, which I was not able to open. I think I ran into the same problem with my own hw; I assume it has to do with what repo the image file is in?
* Nice work!

**Pull request closed without merging**